app/assets/stylesheets/_dashboard.less

```diff
@@ -2,79 +2,166 @@
   margin-top: 0px;
 }
 
-.dataset-thumbnail {
-  img {
-    opacity: 0.8;
-    position: relative;
-    object-fit: cover;
-    height: 200px;
-    width: 100%;
-    display: block;
-  }
-}
+.gallery-dataset-col{
+  paddingBottom: 16px;
 
-.dataset-thumbnail-buttons {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  width: 100%;
-  height: 200px;
 
-  a {
-    box-shadow: white 0px 0px 4px;
-    width: 40px;
-    height: 40px;
-    border-radius: 10px;
-    background: rgba(0, 0, 0, 0.8);
-    display: inline-block;
+  .spotlight-item-card {
+    height: 300px;
+    width: 100%;
     position: relative;
-    margin: 0 3px;
+    paddingLeft: 15px;
+    paddingRight: 15px;
 
-    img {
-      width: 25px;
-      height: 25px;
-      margin: 7.5px;
-      filter: brightness(0) invert(1);
+    @media only screen and (max-width: 600px){
+      height: 700px;
+    }
+
+
+    .ant-card-body{
+      width: 100%;
+      height: 100%;
+
+
+      .dataset-thumbnail { 
+      height: 100%;
```

Comment:
The indentation is a bit off here. If you use sublime, the package JsPrettier should also format less files for you if you want to save yourself some trouble :)